eng/tox/tox.ini

```diff
@@ -159,6 +159,7 @@ deps =
   types-requests==2.31.0.6
   types-six==1.16.21.9
   types-redis==4.6.0.7
+  PyGitHub>=1.59.0
```

Comment:
I don't believe we need this for the non-next mypy env. Is there a reason you added it?

Reply:
copy/pasted will remove